code/stack/model/stack.py

```diff
@@ -0,0 +1,41 @@
+class Stack:
+    def __init__(self, size=None):
+        self.stack = []
+        self.max_size = 100
+        if size is not None:
+            if type(size) is not int:
```

Comment:
maybe `if not isinstance(size, int):` will be better?

Reply:
All done. I think it's looking good, thanks.

Reply:
Good. 👍